.circleci/workflows.yml

```diff
@@ -556,6 +644,41 @@ jobs:
           name: Empty step to make sure the job always has steps
           command: echo
 
+  build-docker-aarch64:
+    description: Job that builds and uploads a Docker image, if needed.
+    docker:
+      - image: cimg/base:2023.06 # Only specific images are supported by ARM https://discuss.circleci.com/t/product-launch-arm-docker-generally-available/48601
```

Comment:
Can we see whether `cimg/base:current` works nowadays? That message in their discussion forum might just be because at the time of the release older images didn't support Arm.

Reply:
Yes!